backend/routers/user.py

```
@@ -0,0 +1,62 @@
+    db: Session = Depends(get_db)
+):
+    update_user_id = user_service.update_user(user_id, user, db)
+    if update_user_id is None:
+        raise HTTPException(status_code=404, detail='User not found')
+    return update_user_id
+
+
+@router.put("/{user_id}", response_model=UserOut)
+def delete_user_id(
+    user_id: int,
+    db: Session = Depends(get_db)
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)